Products/ZenUI3/browser/resources/js/zenoss/HierarchyTreePanel.js

```diff
@@ -666,17 +656,38 @@
                             this.filterTree(field);
                         }
                     }
-                };
+                },
+                rootNode = this.getRootNode();
 
             if (liveSearch) {
                 listeners.change = this.filterTree;
             }
             if (this.searchField && this.ownerCt.regSearchListeners) {
                 this.ownerCt.regSearchListeners(listeners);
             }
-            this.getRootNode().expand(false, true, function (node) {
-                node.expandChildNodes();
-            });
+            // check if root node is loaded and expand it;
+            // if not loaded we should listen store load and than expand root node;
```

Comment:
on afterrender we check if tree root node is loaded and than expand and select first node on expand if there are no cached selection, else we listen store load event and than expand root node and select visible root;